docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -198,11 +198,34 @@ const hardcodedTypeLinks: Record<string, string> = {
   WebGLFramebuffer: 'https://developer.mozilla.org/en-US/docs/Web/API/WebGLFramebuffer',
 };
 
-const renderWithLink = (name: string, type?: string) => {
+const packageLinks: Record<string, string> = {
+  'expo-manifests': 'manifests',
+};
+
+const renderWithLink = ({
+  name,
+  type,
+  typePackage,
+}: {
+  name: string;
+  type?: string;
+  typePackage: string | undefined;
```

Comment:
```suggestion
  typePackage?: string;
```

Typing nit.

Reply:
There's actually surprisingly a difference between the two:
- `typePackage: string | undefined;` requires the key at callsites but an `undefined` value can be passed.
- `typePackage?: string;` makes the key optional at the callsites, or an `undefined` value can be passed.

In this instance, I want the first one since all the callsites should pass the key.